### DISCUSSION TITLE
Launch option to open the app in tray bar

### DISCUSSION THREAD

**Discussion by MadByteDE** [tutao/tutanota#3731] [Ideas]
I'd like to be able to launch the desktop app directly to the tray bar without the GUI opening. 
I'm running Linux and auto-start Tutanota via a sh script. I'd like to do something like:
```sh
tutanota-desktop --silent
```
to skip the GUI popping up.

**Comment by bedhub** [comment]
There are the settings "Run in background" and "Run on startup" in Settings > Desktop which should solve this use case.

**Reply by MadByteDE** [reply]
Nope doesn't do anything for me, is activated since I've installed the app a year ago seemingly. (KDE Plasma btw)

**edit**
To clarify: The app does work in the background and push notifications are sent accordingly, but I'd like the app to not even open the GUI when my system starts but instead just receive notifications if there are new mails. And if there is one waiting for me, I'd just click the notification and read it.

**Comment by johnbotris** [comment]
@MadByteDE Assuming you don't have a DE that supports xdg autostart, you can also run it with `-a` for silent startup

**Reply by MadByteDE** [reply]
Adding `-a` as launch parameter still opens the GUI.

**edit**
I'm on Arch using KDE Plasma if that's useful information to you. Using `tutanota-desktop -a` with zsh shell still opens the GUI first instead of doing a silent start up.
